docs/installation/docker.md

```diff
@@ -86,6 +86,18 @@ Please execute the script itself to view all additional options:
 ./docker-run.sh
 ```
 
+## Running the Docker app with docker-sync
+
+To run the app with [docker-sync](http://docker-sync.io/) please follow the
+following steps:
+
+1. Install `docker-sync`: `gem install docker-sync`
+2. Make sure your system has `rsync` available
```

Comment:
I believe you are missing a `docker-compose build` step here

Reply:
I do indeed. Funny, it definitely was there at one point. Sorry for that!